source/dofs/dof_tools_sparsity.cc

```diff
@@ -1014,12 +1030,13 @@ namespace DoFTools
               dofs_on_this_cell.resize (cell->get_fe().dofs_per_cell);
               cell->get_dof_indices (dofs_on_this_cell);
 
-              // make sparsity pattern for this cell
+              // make sparsity pattern for this cell also taking into account
+              // the couplings due to face contributions on the same cell
               constraints.add_entries_local_to_global (dofs_on_this_cell,
                                                        sparsity,
                                                        keep_constrained_dofs,
-                                                       bool_int_dof_mask[cell->active_fe_index()]);
-              // Loop over all interior neighbors
+                                                       bool_int_and_flux_dof_mask[cell->active_fe_index()]);
+              // Loop over interior faces
```

Comment:
put one of two empty lines above this comment to separate it from the code before.